src/Commands/RemindOperation.php

```diff
@@ -25,24 +25,28 @@ class RemindOperation extends Command
      */
     protected $description = 'Check for operation to be reminded on Slack';
 
-    /**
-     * @var array
-     */
-    private $marks = [15, 30, 60];
-
     /**
      * Process the command.
      */
     public function handle()
     {
-        if (setting('kassie.calendar.slack_integration', true) == 1) {
-            $ops = Operation::all()->take(-50);
-            $now = Carbon::now('UTC');
+        # Ensure we send reminders starting with furthest in the future. That way
+        # when more than one event is being reminded, the last reminder in chat
+        # is the next event to occur.
+        $configured_marks = setting('kassie.calendar.notify_operation_interval', true);
```

Comment:
will not this change also require an update of the schedule seeder -> https://github.com/BenHUET/eveseat-calendar/blob/master/src/database/seeds/ScheduleSeeder.php#L15 ?

Currently, it's not designed to be called every minute.

Reply:
From what I can see, the schedule seeder is not in use. The reminder task is being executed from cron, configured in the service provider: https://github.com/BenHUET/eveseat-calendar/blob/master/src/CalendarServiceProvider.php#L29

Maybe have a separate PR to either migrate complete to the schedule seeder, or remove it?

Reply:
hum, indead - not address by this pull request
you had a nice catch with the provider - that also mean currently, the plugin is calling the command by default - so peoples may have duplicate calls if they also put it into their schedulers config.

Logic would be to use the seeder instead of the provider override - if you got a bit of time to make that switch and update the seeder accordingly ?

Reply:
This PR is already sizeable, and I would like to keep it focused on the changes specifically around notification settings. I am happy to continue the work to update the seeder but I would prefer to do that as a separate PR, and get this one merged sooner.